src/components/Breadcrumb/index.tsx

```diff
@@ -60,7 +60,7 @@ export const BreadCrumb: React.FC<Props> = ({
 
 interface PatchBreadcrumbProps {
   patchAuthor: string;
-  analytics: any;
+  analytics: ReturnType<typeof useBreadcrumbAnalytics>;
```

Comment:
The type for useBreadCrumbAnalytics is established in it's file. Could we import that here instead using ReturnType and typeof?